Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -1996,6 +1999,11 @@ script:
       name: ip
     - description: The endpoint hostname.
       name: hostname
+    - default: true
+      description: ''
+      isArray: true
+      name: endpoint
+      required: true
```

Comment:
Can you please elaborate on this addition? why is it default? why is it required?